integration/client/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/containerd/ttrpc v1.2.1
 	github.com/containerd/typeurl/v2 v2.1.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
+	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221013174636-8159c8264e2e
```

Comment:
@tianon The first commit can be dropped now if you rebase (this was updated in https://github.com/containerd/containerd/pull/8498)